### ISSUE TITLE
Add --css CLI option

### ISSUE THREAD

**Issue by danburzo** [danburzo/percollate#27]
Add a `--css` CLI option to allow sending short style snippets from the CLI directly, without having to use a custom HTML/CSS file. For example, changing the page size:

```bash
percollate --output some.pdf --css "@page { size: A4; }" http://example.com
```

The CSS will be __appended__ to the stylesheet.

**Comment by butu5** [comment]
This option will be cool

**Comment by danburzo** [comment]
Implemented in `percollate@0.2.8`
